epochStart/metachain/systemSCs.go

```diff
@@ -267,35 +273,32 @@ func (s *systemSCProcessor) getValidatorTopUpMap(validators []state.ValidatorInf
 	return ret, nil
 }
 
-func compareByXORWithRandomness(pubKey1, pubKey2, randomness []byte) bool {
-	lenPubKey := len(pubKey1)
+func calcNormRand(randomness []byte, expectedLen int) []byte {
 	lenRand := len(randomness)
-
-	minLen := core.MinInt(lenPubKey, lenRand)
-	maxLen := core.MaxInt(lenPubKey, lenRand)
-	repeatedCt := maxLen/minLen + 1
+	minLen := core.MinInt(expectedLen, lenRand)
```

Comment:
why do you need minLen and maxLen?
you just need randLen and expectedLen

also can you rename lenRand to randLen?

Reply:
Right, this code could be simplified. Changed it